inappmessaging/USERGUIDE.md

```diff
@@ -274,6 +312,15 @@ buildscript {
     }
 }
 ```
+</details>
+
+<details>
+<summary>Getting error 401 (click to expand)</summary>
+
+### Getting error 401
+(For our internal users) If you are encountering error 401 when calling ping/impression endpoint - check your app's scopes list if it contains all required entries OR stop putting RAE Token in `UserInfoProvider` (this breaks user targeting)
```

Comment:
For system troubleshooting I suggested on the iOS PR to just say 
> Rakuten developers experiencing problems should refer to the Troubleshooting Guide on the internal developer documentation portal.

I think we should have the same here.

Reply:
Updated to suggested statement.